__tests__/helpers/LocalStorageMock.js

```diff
@@ -4,14 +4,17 @@ const localStorageMock = (function () {
 
     return {
         getItem(key) {
-            return store[key];
+            return store[key] || null;
```

Comment:
Why the sudden `null`? `Storage` instances return undefined when a key doesn't exist iirc.